src/NzbDrone.Core.Test/Files/Indexers/TorrentRss/ImmortalSeed.xml

```diff
@@ -347,11 +347,11 @@
       <pubDate>2015-02-06 03:13:33</pubDate>
     </item>
     <item>
-      <title>The.Blacklist.S02E10.720p.HDTV.X264-DIMENSION</title>
+      <title>The.Blocklist.S02E10.720p.HDTV.X264-DIMENSION</title>
```

Comment:
Keep these, it’s a legit series name…. Or go ahead and update with a relevant movie related response while in here.

Reply:
Yeah this was my bad, I missed the file exclusion for xml when I was searching for blacklist!